allure-java-commons/src/main/java/io/qameta/allure/util/AspectUtils.java

```diff
@@ -57,6 +73,13 @@ public static String getName(final String nameTemplate,
         return params;
     }
 
+    public static Map<String, Object> getParametersMap(final JoinPoint joinPoint) {
+        final MethodSignature methodSignature = (MethodSignature) joinPoint.getSignature();
+        final Map<String, Object> params = getParametersMap(methodSignature, joinPoint.getArgs());
+        params.put("this", joinPoint.getThis());
```

Comment:
make sure this works as expected in case of static step methods (I guess `this` will be null)

Reply:
fixed